p/scripts/main.js

```diff
@@ -2028,6 +2028,7 @@ function init_normal() {
 }
 
 function init_main_beforeDOM() {
+	history.scrollRestoration = 'manual';
```

Comment:
Makes sense. The sample on MDN does this fwiw:

```js
if (history.scrollRestoration) {
  history.scrollRestoration = 'manual';
}
```

Reply:
It is probably a good idea to use the same construct, then, don't you think?

Reply:
I suppose the goal would be to avoid accidentally fooling some other piece of code. You can safely assign history.just_about_anything.